### PR TITLE
fix(test) : comments test added

### PR DESCRIPTION
**Pull request by Raunak1203**

### CODE REVIEW

File: runtime/src/tests/functionalTests/ui/planner/workitem-quickpreview.ts [hunk 1]
```diff
@@ -91,6 +100,27 @@ export class WorkItemQuickPreview extends ui.BaseElement {
...
 
+  private async addComment(comment: string) {
+    await this.loadingAnimation.untilAbsent();
+    await this.commentsField.clickWhenReady();
+    await this.commentsInputField.sendKeys(comment);
+  }
+
+  async addCommentAndSave(comment: string) {
+    await this.addComment(comment);
+    await this.commentSaveButton.click();
```
Comment (jarifibrahim) [inline runtime/src/tests/functionalTests/ui/planner/workitem-quickpreview.ts]: We can use `this.commentSaveButton.clickWhenReady()` also. This makes sure the button is ready and clickable

File: runtime/src/tests/functionalTests/ui/planner/workitem-quickpreview.ts [hunk 2]
```diff
@@ -91,6 +100,27 @@ export class WorkItemQuickPreview extends ui.BaseElement {
...
+  }
+
+  async addCommentAndCancel(comment: string) {
+    await this.addComment(comment);
+    await this.commentCancelButton.click();
```
Comment (jarifibrahim) [inline runtime/src/tests/functionalTests/ui/planner/workitem-quickpreview.ts]: We can use `this.commentCancelButton.clickWhenReady()` also. This makes sure the button is ready and clickable

File: runtime/src/tests/functionalTests/ui/planner/workitem-quickpreview.ts [hunk 3]
```diff
@@ -91,6 +100,27 @@ export class WorkItemQuickPreview extends ui.BaseElement {
...
+  }
+
+  async getAllComments() {
+    await this.commentsText.length>0;
```
Comment (jarifibrahim) [inline runtime/src/tests/functionalTests/ui/planner/workitem-quickpreview.ts]: How does this work? Do we need this?

File: runtime/src/tests/functionalTests/ui/planner/workitem-quickpreview.ts (hunk 1 above)
Comment (Raunak1203) [reply runtime/src/tests/functionalTests/ui/planner/workitem-quickpreview.ts]: done

File: runtime/src/tests/functionalTests/ui/planner/workitem-quickpreview.ts (hunk 3 above)
Comment (Raunak1203) [reply runtime/src/tests/functionalTests/ui/planner/workitem-quickpreview.ts]: We don't need this... removed

**Pull request merged**